pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala

```diff
@@ -15,28 +15,40 @@ case class MetsData(
   accessConditionStatus: Option[String] = None,
   accessConditionUsage: Option[String] = None,
   fileReferencesMapping: List[(String, FileReference)] = Nil,
-  titlePageId: Option[String] = None
+  titlePageId: Option[String] = None,
+  deleted: Boolean = false
 ) {
 
   def toWork(version: Int,
-             modifiedTime: Instant): Either[Throwable, Work.Invisible[Source]] =
-    for {
-      license <- parseLicense
-      accessStatus <- parseAccessStatus
-      item = Item[IdState.Unminted](
-        id = IdState.Unidentifiable,
-        locations = List(digitalLocation(license, accessStatus)))
-    } yield
-      Work.Invisible[Source](
-        version = version,
-        state = Source(sourceIdentifier, modifiedTime),
-        data = WorkData[DataState.Unidentified](
-          items = List(item),
-          mergeCandidates = List(mergeCandidate),
-          thumbnail = thumbnail(sourceIdentifier.value, license, accessStatus),
-          images = images(version, license, accessStatus)
-        )
-      )
+             modifiedTime: Instant): Either[Throwable, Work[Source]] = {
+    deleted match {
+      case true =>
+        Right(
+          Work.Deleted[Source](
+            version,
+            state = Source(sourceIdentifier, modifiedTime),
+            None))
```

Comment:
Is this the deletion reason? Let's fill it in with some useful info, and put a keyword argument here so the code is easier to read.